### ISSUE TITLE
Make LED 'circle' match the current output color of the LED

### ISSUE THREAD

**Issue by ehamner** [CMU-CREATE-Lab/flutter-app-android#73]
The (currently) green circle on the LED relationship lable should show the current output color of the LED.

![robot-13-sensordata-speaker-mute](https://cloud.githubusercontent.com/assets/24419423/22755735/7211dd7e-ee12-11e6-9eda-d530290b2dba.png)

**Comment by jenncross** [comment]
This could be implemented one of two ways....

Either:
A. Poll the Flutter constant for the output states (along with sensor values) - which seems complex.
B. Use the current known sensor values and the know links, to compute the output states in the app in parallel to the Flutter hardware.


--- Granted in the long term option A. might be better, once all of the link types are implemented.

**Comment by jenncross** [comment]
This seems nearly impossible to achieve accurately ... due to:

1. Maximum polling frequency of the Flutter
2. State-based value for Change & Cumulative relationships
3. Rapidly changing values for Amplitude & Frequency relationships

It seems like this is then only feasible on Proportional & Fixed relationships. It will be extremely confusing to have this work on some but not all relationship types.


_**tl;dr: I suspect that this is not likely to be resolved anytime soon.**_

**Comment by ehamner** [comment]
Replaced by #159, split LED circle to show both colors selected, which we liked when tested in Pilot 5.
